t/api/15-users.t

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env perl
+# Copyright (C) 2017-2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use Test::Most;
+
+use Date::Format;
+use FindBin;
+use lib "$FindBin::Bin/../lib";
```

Comment:
Please move these two lines just before the first *internal* import, i.e. two lines down

Reply:
please do this still, move below the "Test::Warnings" import which is external